kernel/build/config-amd64

```diff
@@ -2483,7 +2537,6 @@ CONFIG_IGB=m
 CONFIG_IGB_HWMON=y
 CONFIG_IGB_DCA=y
 CONFIG_IGBVF=m
-CONFIG_IXGB=m
```

Comment:
```
commit e485f3a6eae0849f83b94936778a2325f72a0c89
Author: Tony Nguyen <anthony.l.nguyen@intel.com>
Date:   Fri Mar 17 13:09:03 2023 -0700

    ixgb: Remove ixgb driver
    
    There are likely no users of this driver as the hardware has been
    discontinued since 2010. Remove the driver and all references to it
    in documentation.
    
    Suggested-by: Jakub Kicinski <kuba@kernel.org>
    Signed-off-by: Tony Nguyen <anthony.l.nguyen@intel.com>
    Acked-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
```

Reply:
i want to see who complains :joy: